### DISCUSSION TITLE
Popups: [Proposal] Add Overlay Color Change Property

### DISCUSSION THREAD

**Discussion by ElegantCatterpillar** [CommunityToolkit/Maui#1990] [New Feature Discussions]
A Popup property to be able to change the color of the overlay, currently it is a transparent Medium color.

In some cases I have needed to make the color transparent, or to make the color stronger or weaker. More so when I created custom components from the Popup.

For example use:
OverlayColor = "Transparent"

<?xml version="1.0" encoding="utf-8" ?>
<toolkit:Popup
    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
    CanBeDismissedByTappingOutsideOfPopup="True"
    OverlayColor="Green"
    Color="Red">
</toolkit:Popup>

**Comment by CliffAgius** [comment]
Closing as Duplicate of #1799
